ru/lessons/libraries/guardian.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: page
+title: Guardian (Основы)
+category: libraries
+order: 1
+lang: ru
+---
+
+[Guardian](https://github.com/ueberauth/guardian) - часто используемая библиотека, основанная на [JWT](https://jwt.io/) (Javascript Web Token).
+
+{% include toc.html %}
+
+## JWT
+
+Что такое JWT? JWT занимается созданием токена для аутентификации. В то время как многие системы аутентификации предоставляют доступ только к идентификатору ресурса, JWT позволяет хранить еще и другую информацию, такую как:
+
+* Кто выдал токен
+* Для кого выдан токен
+* Какие системы должны использовать этот токен
+* На какое время он выдан
+* Каков тип токена
+* Какие права есть у владельца данного токена
+
+Это всё - простые примеры полей в JWT. Технология позволяет добавлять любую дополнительную информацию, которой требует логика приложения. Главное - не забывайте что этот токен должен быть коротким, ведь он должен поместиться в HTTP заголовке.
```

Comment:
`-` → `&mdash;` in both places.

I also thought that we probably could mention the suggested maximum HTTP header length. There is no limit in the [specification](https://tools.ietf.org/html/rfc7230#section-3.1.1) but most of the servers use the default maximum value (~8KB).
/cc @doomspork